umpf

```diff
@@ -197,21 +197,21 @@ usage() {
 	  -v, --version <version>    with tag: overwrite version number [default: 1]
 
 	Commands:
-	  diff <commit-ish>          show patches not in any topic branch (not
+	  diff <umpf|commit-ish>     show patches not in any topic branch (not
 	                             upstream) and patches missing locally
-	  show <commit-ish>          show an useries file from an umpf
+	  show [umpf|commit-ish]     show an useries file from an umpf
```

Comment:
You're using `<>` above and `[]` here. This should be consistent. And probably both because we use `<>` to indicate "replace this with something appropriate" and `[]` for optional arguments and both is relevant here.

The same for the other changes.

Reply:
With "above" you mean where? Is it also optional for 'diff'?

Reply:
It worked out in that specific case; force-pushed the new state of the branch. Also included a whitespace cleanup in the second commit.